.config/tsaoptions.json

```
@@ -1,10 +1,10 @@
-{  
+{
     "instanceUrl": "https://devdiv.visualstudio.com/",
     "template": "TFSDEVDIV",
     "projectName": "DEVDIV",
     "areaPath": "DevDiv\\NET Fundamentals\\Infrastructure\\Arcade\\SDL",
     "iterationPath": "DevDiv",
     "notificationAliases": [ "dnceng@microsoft.com" ],
-    "repositoryName":"xliff-tasks",
-    "codebaseName": "xliff-tasks"
+    "repositoryName":"scenario-tests",
+    "codebaseName": "scenario-tests"
```

```suggestion
    "codebaseName": "scenario-tests",
    "serviceTreeId": "0617f471-4860-4e7d-a272-37416bd4a87d"
```

@ViktorHofer what do you think about this? I also saw it somewhere only